api4/system.go

```diff
@@ -395,6 +396,10 @@ func testS3(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if cfg.FileSettings.AmazonS3SecretAccessKey != "" && strings.ContainsAny(cfg.FileSettings.AmazonS3SecretAccessKey, "*") {
```

Comment:
Might be good to compare this against `model.FAKE_SETTING`

Reply:
thanks @saturninoabril  change to use that